src/components/TabletApp/RightSide/Menu/Menu.tsx

```diff
@@ -0,0 +1,92 @@
+import { Button, Box, Typography, Skeleton } from "@mui/material";
+import React from "react";
+import COLORS from "../../../../constants/CustomColors";
+
+import QuickBook from "../QuickBook/QuickBook";
+import { Link } from "react-router-dom";
+import CalendarMonthIcon from "@mui/icons-material/CalendarMonth";
+
+interface iMenu {
+    RoomName: string;
+    RoomStatus: number;
+}
+
+const Menu = ({ RoomName, RoomStatus }: iMenu) => {
+    const RoomStatusMessage = [
+        "currently available",
+        "Going to be in use soon",
+        "Ocupied",
+    ];
+    const subText = [
+        "Feel free to book a meeting",
+        "Maybe you have enough time for a quick meeting",
+        "A meeting is already in progress, please wait",
+    ];
+
+    return (
+        <Box
+            sx={{
+                height: "100%",
+                width: "100%",
+                display: "flex",
+                flexDirection: "column",
+                alignItems: "center",
+                paddingTop: "20%",
+            }}
+        >
+            {RoomName != "" ? (
```

Comment:
- same here with ternary expression
- you can move the ternary expression inside <Box> because the wrapper is the same for every branch